base/main_test_bucket_pool.go

```diff
@@ -121,6 +121,7 @@ func NewTestBucketPoolWithOptions(ctx context.Context, bucketReadierFunc TBPBuck
 		unclosedBuckets:        make(map[string]map[string]struct{}),
 		useExistingBucket:      TestUseExistingBucket(),
 		useDefaultScope:        options.UseDefaultScope,
+		skipMobileXDCR:         true, // do not set up enableCrossClusterVersioning until Sync Gateway 4.x
```

Comment:
Think this is wrong for this branch

Reply:
In actual fact below is some code that will overwrite it I think so should be fine

Reply:
Fixed just in case for clarity.